api/parts/mgmt/app_users.js

```diff
@@ -763,16 +764,17 @@ usersApi.export = function(app_id, query, params, callback) {
                 }
             }).then(function() {
                 //export data from metric_changes
-                return run_command('mongoexport ' + dbstr + ' --collection metric_changes' + app_id + ' -q \'{uid:{$in: ["' + res[0].uid.join('","') + '"]}}\' --out ' + export_folder + '/metric_changes' + app_id + '.json');
+                return run_command('mongoexport', [...dbargs, "--collection", "metric_changes" + app_id, "-q", '{uid:{$in: ["' + res[0].uid.join('","') + '"]}}', "--out", export_folder + "/metric_changes" + app_id + ".json"]);
             }).then(function() {
                 //export data from app_users
-                return run_command('mongoexport ' + dbstr + ' --collection app_users' + app_id + ' -q \'{uid: {$in: ["' + res[0].uid.join('","') + '"]}}\' --out ' + export_folder + '/app_users' + app_id + '.json');
+                return run_command('mongoexport', [...dbargs, "--collection", "app_users" + app_id, "-q", '{uid:{$in: ["' + res[0].uid.join('","') + '"]}}', "--out", export_folder + "/app_users" + app_id + ".json"]);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://app.codacy.com/app/ar2rsawseen/countly-server/pullRequest?prid=4577914)